### PR TITLE
Portuguese version of readme.md

### PR DESCRIPTION
**Pull request by Monotox**
I worked on the translations into the Portuguese language as explained [here.](https://github.com/ipfs/ipfs/issues/306#issuecomment-493799850)

### CODE REVIEW

File: translations/pt_BR/README.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+O IPFS está se tornando um novo grande subsistema da Internet. Se construído corretamente, pode complementar ou substituir o HTTP. Pode complementar ou substituir. Parece loucura. _É_ uma loucura.
+
+[![Alpha Demo](../../img/ipfs-alpha-video.png)](https://www.youtube.com/watch?v=8CMxDNuuAiQ)
+
+Quer ver mais? Confira a palestra de Juan Benet no Sourcegraph - [IPFS: The Permanent Web](https://www.youtube.com/watch?v=Fa4pckodM9g).
+
+### Sumário
+
+O IPFS é um protocolo que:
```
Comment (momack2) [inline translations/pt_BR/README.md]: As a non-speaker - is the leading "O" for stylistic or linguistic purposes? (aka does "IPFS é um protocolo que" mean "IPFS is a protocol that:" - or is it ungrammatical?

File: translations/pt_BR/README.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+**O IPFS é um trabalho em andamento!**
+
+Observe que o IPFS é um trabalho em andamento. É um plano ambicioso para tornar a Internet mais livre, aberta, segura e de alto desempenho. Baseia-se nas boas ideias de numerosos sistemas distribuídos testados em campo.
+
+Hoje, há uma implementação principal do protocolo IPFS (em [Go](https://github.com/ipfs/go-ipfs)) com [mais projetos a caminho](#protocol-implementations) ([JavaScript](https://github.com/ipfs/js-ipfs) e [Python](https://github.com/ipfs/py-ipfs)).
+
+### Distribuição Alpha
+
+In February of 2015, the [go-ipfs](https://github.com/ipfs/go-ipfs) implementation was released as an "Alpha Distribution". Since then, go-ipfs has been making regular releases on the road towards Beta. Both [js-ipfs](https://github.com/ipfs/js-ipfs) and [py-ipfs](https://github.com/ipfs/py-ipfs) are in progress.
```
Comment (momack2) [inline translations/pt_BR/README.md]: This seems to be duplicated - once in english and then the version in Portuguese

File: translations/pt_BR/README.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+|               | https://github.com/digitalkaoz/php-ipfs-api               | ?? |
+| C#            | https://github.com/jeremy-ellis-tech/net-ipfs-http-client | ?? |
+|               | https://github.com/richardschneider/net-ipfs-api          | ?? |
+| C++           | https://github.com/vasild/cpp-ipfs-api                    | ?? |
+| Erlang        | https://github.com/hendry19901990/erlang-ipfs-http-client | ?? |
+| Objective C   | N/A                                                       | ?? |
+| Julia         | contato: [@rened](https://github.com/rened)               | ?? |
+| Lua           | contato: [@seclorum](https://github.com/seclorum)         | ?? |
+
+Please help by contributing to one of the above client libraries. If you would like to create another, please see the [IPFS HTTP Client Implementation Guide](https://github.com/ipfs/go-ipfs/blob/master/docs/implement-api-bindings.md), and [tell us](https://github.com/ipfs/ipfs/issues/83) so we can help!
```
Comment (momack2) [inline translations/pt_BR/README.md]: also has english section

File: translations/pt_BR/README.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+Por favor ajude contribuindo para uma das bibliotecas acima. Caso você queira implementar em outra linguagem, consulte o [Guia de Implementação do Cliente HTTP IPFS](https://github.com/ipfs/go-ipfs/blob/master/docs/implement-api-bindings.md) e [avise-nos](https://github.com/ipfs/ipfs/issues/83) para que possamos ajudar!
+
+### Diretório de Projetos
+
+This aims to be a directory of all the various repos in the [IPFS Github Organization](https://github.com/ipfs), and other closely related things.
```
Comment (momack2) [inline translations/pt_BR/README.md]: and here

**Comment by momack2** [comment]
Thank you so much for the translation @Monotox!! This is awesome! 

Adding in @daviddias and @hugomrdias who might have translation suggestions. I just made a few comments on duplicate english sections with the following PT section. =]

File: translations/pt_BR/README.md (hunk 1 above)
Comment (Monotox) [reply translations/pt_BR/README.md]: "O" is for linguistic purposes. 

Everything is "he" or "she" in Portuguese. "O" is equal to "he", so, he (IPFS) is an protocol.

Comment (momack2) [reply translations/pt_BR/README.md]: Ok - just checking. I saw it for some bullets but not others, so was a little confused...

**Comment by hugomrdias** [comment]
@momack2 i would prefer for other native pt-br speakers to review this, because in pt-pt some parts of the text would be "wrong" but would make text harder to read for pt-br readers.

**Comment by Monotox** [comment]
> Muito massa! Obrigado @Monotox!
> 
> Estou curioso para conhecer a comunidade do IPFS no Brasil. Já existe algum meetup? Posso ajudar a criar a conta no meetup.com de São Paulo -- [ipfs/community#261](https://github.com/ipfs/community/issues/261) -- e quem sabe, visitar em breve :D
> 
> Valeu!

Valeu, David! 

Não conheço iniciativas de meetup sobre IPFS em São Paulo. Pra te falar a verdade, estou bem surpreso de que tenha um número considerável de pessoas envolvidas da comunidade brasileira.

Caso esteja disposto a organizar ou até mesmo participar, super topo! Basta entrar em contato.

**Pull request merged**